python/python/tests/test_transaction.py

```diff
@@ -1043,6 +1043,35 @@ async def test_transaction_object_commands(
         finally:
             await glide_client.config_set({maxmemory_policy_key: maxmemory_policy})
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_transaction_xinfo_stream(
+        self, glide_client: TGlideClient, cluster_mode: bool, protocol
+    ):
+        key = get_random_string(10)
+        stream_id1_0 = "1-0"
+        transaction = ClusterTransaction() if cluster_mode else Transaction()
+        transaction.xadd(key, [("foo", "bar")], StreamAddOptions(stream_id1_0))
+        transaction.xinfo_stream(key)
+        transaction.xinfo_stream_full(key)
+
+        response = await glide_client.exec(transaction)
+        assert response is not None
+        # transaction.xadd(key, [("foo", "bar")], StreamAddOptions(stream_id1_0))
```

Comment:
Do you want to uncomment?

Reply:
oh, this comment was meant to clarify which command was executed to get this specific response in the transaction response array, similar to what we do on the java side. The comment is too long for me to put right beside the assertion. I can see how this would be confusing though, maybe I'll remove the comments since its a short test anyways